src/config/dynamic-theme-fixes.config

```diff
@@ -4147,6 +4147,16 @@ CSS
 
 ================================
 
+streamable.com
+
+INVERT
+.landing .features-pane .feature span svg
```

Comment:
Just to be sure with element you can use >
`.feature > span > svg`
You can do the same for img on line 4155.as elements are risky when using in this way.